pom.xml

```diff
@@ -85,8 +85,8 @@
 
             <plugin>
                 <groupId>com.google.appengine</groupId>
-                <artifactId>appengine-maven-plugin</artifactId>
-                <version>${appengine.target.version}</version>
+                <artifactId>gcloud-maven-plugin</artifactId>
+                <version>2.0.9.88.v20151125</version>
```

Comment:
@loosebazooka I think the gcloud plugin updates pretty much independently from other libraries. Keeping it here makes it easier for us (Cloud DevRel) to update all our samples with a script when there is a gcloud update.